frontend/src/index.pug

```diff
@@ -56,7 +56,7 @@ html
           #tab-resize(onmousedown="registerMouseMove()")
           .tab-close(v-on:click="deactivateTab")
             i.fas.fa-caret-right
-          .tab-content.tab-padding
+          .tab-content.panel-padding
```

Comment:
can you justify the reason for creating a new `panel-padding` instead of using the `tab-padding`?

Reply:
`tab-padding` was created by me in PR #887 to make the paddings for the top and bottom equal. But the left and right paddings are still not equal, as brought forward by @0blivious . This is due to the fact that the `divider` width eats up into the left padding for the right panel. Therefore I need to make the left padding for the right panel different than the rest. This is why I created the `panel-padding` css. Later on this can be refactored to `panels.css`

Reply:
alright noted